src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: 706;
```

Comment:
this value shouldn't be hardcoded. use calculation based on cell width and gap between them